### ISSUE TITLE
Question re getting an index

### ISSUE THREAD

**Issue by miloblum** [rjaros/kvision#177]
I'm a long time java server side developer attempting to build a front end for the first time.  I'm new to Kotlin, KVision, Javascript and HTML.  So basically this is a big learning experience for me.  I've taken your examples, which are great, and have adapted them to my use case to get something up and running.  One thing I am struggling with is how to get the index of a selected value in any sort of a select list.  I can get the currently selected value no problem, but I can't figure out a way to get the currently selected index without searching the list for the value.  The javascript and html I have looked at all refer to a property called "selectedIndex" but I don't see this exposed anywhere in kvision.  Hopefully this is a simple/stupid question with a quick answer.  Thanks for your help.

**Comment by rjaros** [comment]
Hello and thx for your feedback :-)

At the moment there is no specific KVision API to get selected index for a select component. It should be, so I'm leaving this issue open as a feature request.

For now, you can get the index using dynamic JavaScript or jQuery code like this:
```
            val sel = simpleSelectInput(listOf("a" to "Option 1", "b" to "Option 2", "c" to "Option 3"))
            button("Show index").onClick {
                console.log(sel.getElement().asDynamic().selectedIndex)
                console.log(sel.getElementJQuery()?.prop("selectedIndex"))
            }
            val sel2 = simpleSelect(listOf("a" to "Option 1", "b" to "Option 2", "c" to "Option 3"))
            button("Show index 2").onClick {
                console.log(sel2.input.getElement().asDynamic().selectedIndex)
                console.log(sel2.input.getElementJQuery()?.prop("selectedIndex"))
            }
```
The same should work for `SelectInput` and `Select` component.

**Comment by miloblum** [comment]
Thank you!  Very glad I’m not losing my mind.  Appreciate the quick
response and thank you for creating and supporting kvision.

**Comment by rjaros** [comment]
`selectedIndex` property is available for all select components in version 3.13.1
